src/commands/list.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies.
+ */
+const tableImport = require( 'table' );
+const { table } = tableImport;
```

Comment:
This can be made simpler:

```diff
- const tableImport = require( 'table' );
- const { table } = tableImport;
+ const { table } = require( 'table' );
```